nmagent/responses.go

```diff
@@ -37,8 +43,74 @@ type NCVersionList struct {
 	Containers []NCVersion `json:"networkContainers"`
 }
 
+// HomeAZFix is an indication that a particular bugfix has been applied to some
+// HomeAZ.
+type HomeAZFix int
+
+func (h HomeAZFix) String() string {
+	switch h {
+	case HomeAZFixInvalid:
+		return "HomeAZFixInvalid"
+	case HomeAZFixIPv6:
+		return "HomeAZFixIPv6"
+	default:
+		return "Unknown HomeAZ Fix"
+	}
+}
```

Comment:
@timraymond / @ZetaoZhuang Do we need this function? Don't think we're using this anywhere.

Reply:
Yes, this is called by the `fmt` package using reflection. Otherwise, it's an int with no meaning in logs.

Reply:
I see, thanks for the explanation. I didn't know about reflection, so I read up on it.